ivy/functional/ivy/general.py

```diff
@@ -3263,6 +3264,53 @@ def inplace_update(
 inplace_update.unsupported_dtypes = {"torch": ("bfloat16",)}
 
 
```

Comment:
We should set a default value to the mode like [here](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/functional/ivy/general.py#L379)
We should also be correctly passing the mode correctly to the [`ivy/__init__.py`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/__init__.py) file [here](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/__init__.py#L937) and [here](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/__init__.py#L1416)

Reply:
We'd also need to add a `unset_inplace_mode` method like the other modes